packages/sdk-cli/src/models/localRelay/relayInfo.test.ts

```diff
@@ -141,25 +148,30 @@ describe('pollRelayInfo', () => {
       Math.floor(timeout / pollExecTime),
     );
 
-    readJsonFileSpy.mockReset();
+    readJsonFileSpy.mockRestore();
   });
 
   /**
    * First poll request's execution time 2000ms exceeds the total specified 1000ms timeout.
    */
   it('rejects on timeout', async () => {
+    jest.useFakeTimers();
+
     const timeout = 1000;
     const readJsonFileSpy = jest.spyOn(util, 'readJsonFile').mockImplementation(
       () =>
         new Promise<false>((resolve) => setTimeout(() => resolve(false), 2000)),
     );
 
-    await expect(pollRelayInfo(timeout, undefined)).rejects.toHaveProperty(
+    const poll = pollRelayInfo(timeout, undefined);
+    jest.runAllTimers();
+    await expect(poll).rejects.toHaveProperty(
       'message',
       `Timed out after waiting for ${timeout} ms`,
     );
 
-    readJsonFileSpy.mockReset();
+    readJsonFileSpy.mockRestore();
+    jest.useRealTimers();
```

Comment:
This will leak if any of the lines above fail and keep fake timers for other tests. Reset in an afterEach (it's a no-op if real timers are already being used iirc).